test/objects.js

```diff
@@ -775,11 +775,12 @@
     assert.ok(_.has(obj, 'func'), 'has() works for functions too.');
     obj.hasOwnProperty = null;
     assert.ok(_.has(obj, 'foo'), 'has() works even when the hasOwnProperty method is deleted.');
-    var child = {};
-    child.prototype = obj;
+    var child = _.create(obj);
     assert.ok(!_.has(child, 'foo'), 'has() does not check the prototype chain for a property.');
     assert.strictEqual(_.has(null, 'foo'), false, 'has() returns false for null');
     assert.strictEqual(_.has(void 0, 'foo'), false, 'has() returns false for undefined');
+    child = _.create(null, {1: 'foo'});
```

Comment:
Test it out. `_.create` has an `isObject` check. The `null` behavior is not shimmable so not supported in Underscore's `_.create` method.